src/gateway/app.py

```
@@ -0,0 +1,244 @@
+            self._audit_logger.info(
+                "gateway.request",
+                extra={"method": request.method, "path": request.path, "client_id": request.client_id},
+            )
+            try:
+                response = handler(request)
+            except Exception as exc:  # pragma: no cover - safety net
+                self._audit_logger.exception(
+                    "gateway.error",
+                    extra={"method": request.method, "path": request.path, "client_id": request.client_id},
+                )
+                raise exc
```

Ruff (0.14.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)